woudc_data_registry/models.py

```diff
@@ -1129,16 +1153,21 @@ def __init__(self, dict_):
         self.description_en = dict_['description_en']
         self.description_fr = dict_['description_fr']
 
-        self.set_keywords_en(dict_['keywords_en'])
-        self.set_keywords_fr(dict_['keywords_fr'])
+        keywords_column = ['keywords_en', 'keywords_fr']
+        if set(keywords_column).issubset(dict_.keys()):
+            print(f"dict_['keywords_en']: {dict_['keywords_en']}")
```

Comment:
```suggestion
```
Perhaps this can be changed to a `LOGGER.debug()` instead? Otherwise might not be needed.